test/integration/kubernetes_deploy_test.rb

```diff
@@ -64,16 +64,16 @@ def test_pruning_works
     hello_cloud.refute_web_resources_exist
 
     expected_pruned = [
-      'configmap "hello-cloud-configmap-data"',
-      'pod "unmanaged-pod-',
-      'service "web"',
-      'resourcequota "resource-quotas"',
-      'deployment(\.extensions)? "web"',
-      'ingress(\.extensions)? "web"',
-      'daemonset(\.extensions)? "ds-app"',
-      'statefulset(\.apps)? "stateful-busybox"',
-      'job(\.batch)? "hello-job"',
-      'poddisruptionbudget(.policy)? "test"',
+      'configmap[ \/]"?hello-cloud-configmap-data"?',
```

Comment:
Can you make a method like `prune_matcher(kind, group, name)` since we need to construct this format so many times?

Reply:
👍